C/plugins/common/http_sender.cpp

```diff
@@ -25,3 +30,37 @@ HttpSender::HttpSender()
 HttpSender::~HttpSender()
 {
 }
+
+/**
+ * @brief Constructs the file path for the OMF log based on environment variables.
+ *
+ * This function checks for the existence of two environment variables:
+ * FLEDGE_DATA and FLEDGE_ROOT. It constructs the file path to the OMF log
+ * file accordingly. The priority is given to FLEDGE_DATA. If neither
+ * environment variable is set, an error message is printed, and an empty
+ * string is returned.
+ *
+ * @return A string representing the path to the OMF log file, or an empty
+ *         string if neither environment variable is set.
+ */
+std::string HttpSender::getOMFTracePath() 
+{
+    const char* fledgeData = getenv("FLEDGE_DATA");
```

Comment:
Are these environment variables defined if Fledge is installed in its default location, that is, _/usr/local/fledge_? There is a subroutine in _utils.h_ called _getDataDir()_ that does all the logic for you.